(node:3805762) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-nosession-42228',
  containerIp: '10.0.3.211',
  mitmPort: 42228,
  autoAccessFixed: 1
}
2024-11-08 08:59:39.872 - [Info] MITM Version: 2
2024-11-08 08:59:39.874 - [Info] Auto Access Enabled: true
2024-11-08 08:59:39.874 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-08 08:59:39.884 - [Info] SSH man-in-the-middle server for 10.0.3.211 listening on 10.0.3.1:42228
2024-11-08 09:00:31.185 - [Debug] [Connection] Attacker connected: 119.160.192.75 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 09:00:33.067 - [Debug] [Auth] Attacker 119.160.192.75 trying to authenticate with 'password'
2024-11-08 09:00:33.069 - [Debug] [Auto Access] Attacker: 119.160.192.75, Threshold: 1, Attempts: 1
2024-11-08 09:00:33.069 - [Debug] [Auto Access] Compromising the honeypot
2024-11-08 09:00:33.069 - [Debug] [Auto Access] Adding the following credentials: 'auraj:auraj'
2024-11-08 09:00:33.411 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-08 09:00:33.413 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-08 09:00:33.413 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.211
2024-11-08 09:00:33.510 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-08 09:00:34.814 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-08 09:00:36.725 - [Debug] [Connection] Attacker connected: 119.160.192.75 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 09:00:38.502 - [Debug] [Auth] Attacker 119.160.192.75 trying to authenticate with 'password'
2024-11-08 09:00:38.502 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-08 09:00:38.503 - [Debug] [Auth] Attacker: 119.160.192.75 has so far made 1 attempts. Remaining: 5 attempts
2024-11-08 09:00:39.152 - [Debug] [Connection] Attacker closed the connection
2024-11-08 09:00:40.473 - [Debug] [Connection] Attacker connected: 119.160.192.75 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
